### PR TITLE
Feat/aws bucket sync

### PR DESCRIPTION
**Pull request by giangbui**

### CODE REVIEW

File: gen3/bin/kube-setup-secrets.sh [hunk 1]
```diff
@@ -208,7 +208,25 @@ if [[ -f "${WORKSPACE}/${vpc_name}_output/creds.json" ]]; then  # update secrets
...
+  fi
+
+  if ! g3kubectl get configmaps/data-service-manifest > /dev/null 2>&1; then
+    if [[ ! -f "./apis_configs/manifest" ]]; then
+      touch "./apis_configs/manifest"
+    fi
+    g3kubectl create configmap data-service-manifest --from-file=manifest=./apis_configs/manifest
+  fi
+
   # Create the 'sheepdog' and 'peregrine' postgres user if necessary
```
Comment (frickjack) [inline gen3/bin/kube-setup-secrets.sh]: We don't need these secrets and configurations in most commons, so we should only create them if necessary.  Move all these confile file to be under one directory - `apis_configs/dcf_dataservice/creds.json` and `apis_configs/dcf_dataservice/manifest`, and wrap this whole block in an `if` block:
```
if [[ -d apis_configs/dcf_dataservice ]]; then ...
```

File: kube/services/jobs/aws-bucket-replicate-job.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            secretName: "dcf-dataservice-json-secret"
+      containers:
+      - name: awsreplicate
+        image: quay.io/cdis/dcf-dataservice:feat_data_replicate_service
+        imagePullPolicy: Always
+        env:
+          - name: GDC_BUCKET_NAME
+            GEN3_GDC_BUCKET_NAME
+          - name: LOG_BUCKET
+            GEN3_LOG_BUCKET
```
Comment (frickjack) [inline kube/services/jobs/aws-bucket-replicate-job.yaml]: What is the LOG_BUCKET for?

File: kube/services/jobs/aws-bucket-replicate-job.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# run with 
+# g3k runjob aws-bucket-sync \
+# GDC_BUCKET_NAME a_bucket_name \
+# CHUNK_SIZE chunk_size
+#
+# folow the below link  to setup bucket owner grant access to other aws account 
+# https://docs.aws.amazon.com/AmazonS3/latest/dev/example-walkthroughs-managing-access-example2.html
+#
+# setup bucket policy so that the user test can access to the bucket (GDC bucket)
```
Comment (frickjack) [inline kube/services/jobs/aws-bucket-replicate-job.yaml]: Would you give some more context here or in jobs/README.md?
Ex - this job is used to copy objects from GDC to an AWS bucket using a manifest ... something like that?

File: kube/services/jobs/aws-bucket-replicate-job.yaml (hunk 2 above)
Comment (giangbui) [reply kube/services/jobs/aws-bucket-replicate-job.yaml]: To save all the uuid which can not be copied.

File: kube/services/jobs/aws-bucket-replicate-job.yaml (hunk 3 above)
Comment (giangbui) [reply kube/services/jobs/aws-bucket-replicate-job.yaml]: fixed at e9fdab74ac2aa33db7836eb27afa2890695ce597

File: gen3/bin/kube-setup-secrets.sh (hunk 1 above)
Comment (giangbui) [reply gen3/bin/kube-setup-secrets.sh]: fixed at e9fdab74ac2aa33db7836eb27afa2890695ce597

**Pull request merged**